packages/toolpad-core/src/types/index.ts

```diff
@@ -0,0 +1,2 @@
+export type * from './Branding.types';
```

Comment:
Why the .types.ts extension? And is it really necessary to create a separate top level folder for types? I feel like in core/X types are grouped logically with their components.

I feel like it would make most sense to start like MUI core, where all components are exported top level. And we can start organizing them once we're reaching a sizable amount. wdyt?